zeppelin-web/src/app/helium/helium.controller.js

```diff
@@ -149,37 +149,55 @@ export default function HeliumCtrl($scope, $rootScope, $sce,
     return license;
   }
 
-  $scope.enable = function(name, artifact, type, groupId) {
+  const getHeliumTypeText = function(type) {
+    if (type === HeliumType.VISUALIZATION) {
+      return `<a href="https://zeppelin.apache.org/docs/latest/development/writingzeppelinvisualization.html">${type}</a>`; // eslint-disable-line max-len
```

Comment:
Sorry for my late comment. Can we make this link point to `https://zeppelin.apache.org/docs/zeppelin_version/development/writingzeppelinvisualization.html` instead of `latest`? Please see line number 165-166 in this file. 
And also it needs `target=_blank`.